apps/tailwind-components/pages/input/Int.story.vue

```diff
@@ -0,0 +1,20 @@
+<template>
+  <InputTestContainer
+    show-placeholder
+    show-state
+    v-slot="{ placeholder, valid, invalid, disabled }"
+  >
+    <InputInt
```

Comment:
It would be good to add a label here as well (see comment in decimal story)

Reply:
see my comment too